src/Compiler/SyntaxTree/LexFilter.fs

```diff
@@ -1969,7 +1972,8 @@ type LexFilterImpl (
         | MODULE, _ :: _ ->
             insertComingSoonTokens("MODULE", MODULE_COMING_SOON, MODULE_IS_HERE)
             if debug then dprintf "MODULE: entering CtxtModuleHead, awaiting EQUALS to go to CtxtSeqBlock (%a)\n" outputPos tokenStartPos
-            pushCtxt tokenTup (CtxtModuleHead (tokenStartPos, token, NotLexingModuleAttributes))
+            let isNested = match offsideStack with | [ CtxtSeqBlock _ ] -> false | _ -> true
```

Comment:
With that, is

```namespace N

module M =```

also nested?

Reply:
Correct, that's the terminology from the `SyntaxTree`:
https://github.com/dotnet/fsharp/blob/e267bb9f8d590feed1b94b469d78cfce61afecad/src/Compiler/SyntaxTree/SyntaxTree.fs#L1461